locales/en-US/strings.properties

```diff
@@ -115,6 +115,19 @@ time_label_day={number}d
 # in English, while "Home" should be localized matching the about:preferences
 # sidebar mozilla-central string for the panel that has preferences related to
 # what is shown for the homepage, new windows, and new tabs.
+prefs_new_windows_tabs_header=New Windows and Tabs
+prefs_new_windows_description=Choose what you see when you open your homepage, new windows, and new tabs.
+
+prefs_homepage_choice_label=Homepage and new windows:
+prefs_newtab_choice_label=New tabs:
+prefs_homepage_newtab_choice_1_firefox_home=Firefox Home
+prefs_homepage_newtab_choice_2_custom=Custom URL(s)
```

Comment:
Do we really need the (s)?

Reply:
In this case, users can enter more than one, so I think it's needed?

Reply:
We currently don't have [any UI](https://transvision.mozfr.org/?recherche=%28s%29&repo=gecko_strings&sourcelocale=en-US&locale=en-US&search_type=strings_entities) using `(s)` in Firefox (/browser). Maybe just use `Custom URLs…` would work?

Reply:
Ok, that sounds find to me